src/components/CreateTemplateFromTemplate.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from "react";
+import { Navigate, useLocation, useParams } from "react-router-dom";
+import { useCreateTemplateFromTemplate } from "../queries/template-queries";
+import { useAppServices } from "./AppServicesContext";
+import { LoadingScreen } from "./LoadingScreen";
+
+export const CreateTemplateFromTemplate = () => {
+  const { idTemplate } = useParams() as Readonly<{
+    idTemplate: string;
+  }>;
+
+  const { search } = useLocation();
+  const {
+    window,
+    appConfiguration: {
+      dopplerExternalUrls: { templates: templatesUrl },
+    },
+  } = useAppServices();
+  const { mutate, isIdle, isLoading, isSuccess, data, error } =
+    useCreateTemplateFromTemplate();
+
+  useEffect(() => {
+    mutate({ baseTemplateId: idTemplate });
+  }, [mutate, idTemplate]);
+
+  if (isSuccess && data?.success && data?.value?.newTemplateId) {
+    const templateUrl = `/templates/${data?.value?.newTemplateId}${search}`;
+    return <Navigate to={templateUrl} replace={true} />;
+  }
+
+  if (!isIdle && !isLoading) {
+    window.console.error(
+      "Error creating template from template",
+      error || data
+    );
+    window.location.href = templatesUrl;
+  }
```

Comment:
@Cromeror this block is different from the [previous PR](https://github.com/FromDoppler/doppler-editors-webapp/pull/374/files#diff-20c56ac378840196633441d1f69515bdd51e49487ed33fb66850ffe8219c0922R26-R36) because when it fails, the template does not exist, so, I cannot redirect to edit it.

I am logging the error and then redirecting to the Templates section. Since we do not have a good error page or control, or even error messages, I think that it is an acceptable behavior, what do you think?

Reply:
I think that we can´t do anything now because before we never catch these error to this detail level, maybe we could start to do this now but, I don´t know how much beneffic this we can give us. So now for me is fine you implementation.